### PR TITLE
doc(collaset): introduce CollaSet Replace update policy

### PR DESCRIPTION
**Pull request by wu8685**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [x] N
- [ ] Y 

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [x] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [x] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/operating/manuals/collaset.md [hunk 1]
```diff
@@ -436,4 +434,125 @@ NAME                     READY   STATUS    RESTARTS     AGE
...
+metadata:
+  name: collaset-sample
+spec:
+  ...
+  updateStrategy:
+    podUpgradePolicy: Replace  # Options: InPlaceIfPossible, ReCreate, Replace
+```
+
+The `Replace` policy indicates that CollaSet should update a Pod by creating a new one to replace it. 
+Unlike the `Recreate` policy, which deletes the old Pod before creating a new updated one, or the `InPlaceIfPossible` policy, which updates the current Pod in place,
```
Comment (Eikykun) [inline docs/operating/manuals/collaset.md:455]: `Recreate` or `ReCreate`?  In operating defined `CollaSetRecreatePodUpdateStrategyType PodUpdateStrategyType = "ReCreate"`. But I believe the correct way to write it is `Recreate`.
I

**Review by Eikykun — COMMENTED** [review]
PATL

File: docs/operating/manuals/collaset.md (hunk 1 above)
Comment (Eikykun) [inline docs/operating/manuals/collaset.md:455]: `Recreate` or `ReCreate`?  In operating defined `CollaSetRecreatePodUpdateStrategyType PodUpdateStrategyType = "ReCreate"`. But I believe the correct way to write it is `Recreate`.

File: docs/operating/manuals/collaset.md [hunk 2]
```diff
@@ -436,4 +434,125 @@ NAME                     READY   STATUS    RESTARTS     AGE
...
+collaset-sample-vqccr   1/1     Terminating   0          71s
+collaset-sample-nbl6t   0/1     ContainerCreating   0          0s
+collaset-sample-nbl6t   0/1     ContainerCreating   0          0s
+collaset-sample-nbl6t   1/1     Running             0          0s
+collaset-sample-nbl6t   1/1     Running             0          0s
+collaset-sample-vqccr   1/1     Terminating         0          72s
+collaset-sample-vqccr   0/1     Terminating         0          73s
+collaset-sample-vqccr   0/1     Terminating         0          73s
+collaset-sample-vqccr   0/1     Terminating         0          73s
+```
```
Comment (Eikykun) [inline docs/operating/manuals/collaset.md:517]: I don't think using `-w` in the documentation is a good way to demonstrate.
This looks easier to understand: 
```
$ kubectl -n default get pod
collaset-sample-vqccr   1/1     Terminating   0          71s
collaset-sample-nbl6t   0/1     ContainerCreating   0          0s
```

File: docs/operating/manuals/collaset.md (hunk 1 above)
Comment (wu8685) [reply docs/operating/manuals/collaset.md]: Also fix in this PR

**Review by Eikykun — APPROVED** [review]
/lgtm

**Pull request merged**